.circleci/config.yml

```diff
@@ -3,22 +3,16 @@ jobs:
   build:
     docker:
       - image: circleci/golang:1.11.5
-    working_directory: /go/src/github.com/student-kyushu/frau
```

Comment:
What is the reason for erasing this line?

Reply:
This is a mistake, I fixed.